pkg/sql/logictest/testdata/logic_test/row_level_security

```diff
@@ -5930,4 +5930,61 @@ DROP TABLE accounts, non_rls;
 statement ok
 DROP USER alice;
 
+statement ok
+RESET SESSION row_security;
+
+subtest end
+
+# Regression test for #158154. Don't add transitive column dependencies to
+# routines that build RLS constraints.
+subtest regression_158154
+
+statement ok
+CREATE TABLE t158154 (c0 INT, c1 TEXT DEFAULT 'foobarbaz');
```

Comment:
Tests are failing due to:

```
pkg/sql/logictest/testdata/logic_test/row_level_security:5964: 
        expected success, but found
        (0A000) unimplemented: multi-column-family check constraints are not yet supported under read committed isolation
        mutation_builder.go:1193: in addCheckConstraintCols()
```

Locking the column families for this table should do the trick.